daml-lf/transaction/src/stable/protobuf/com/daml/lf/value.proto

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
I was looking at diffs and the dev version of transaction has
```
> // * dev (special staging area for the next version to be released)
```
while the dev version of value doesn’t have that header. maybe worth adding? or maybe drop it from both so stabilizing is just copying the file. Seems easy to forget to drop that header.

Reply:
In general, we cannot  just "copy" the files. 
Since `dev` features may be dropped without further notice, we have to remove from the file any `dev` feature which is not 
stable at the time of freeze. 

I think it is more clean to make a manual pass over the file and remove all reference to dev before freeze.
This could be describe in the document you asked for below.

Reply:
makes sense, then I think we should also add it to `value.proto`.

Reply:
210c843c